src/panels/calendar/dialog-calendar-event-editor.ts

```diff
@@ -108,7 +121,7 @@ class DialogCalendarEventEditor extends LitElement {
     return html`
       <ha-dialog
         open
-        @closed=${this._close}
+        @closed=${this.closeDialog}
```

Comment:
The method is going to run twice: Once inside the methods below, after an action is taken, and once here, after the closed event.

Reply:
Yes, added a bailout to the function